beets/importer.py

```diff
@@ -170,6 +170,22 @@ def history_get():
     return state[HISTORY_KEY]
 
 
+def calculate_quality_score(item, is_album):
```

Comment:
For this and the other utility function below, it would be great to add a docstring at some point (whenever you're ready) to help explain the role of the function to other developers.